Tests/Samples/blend_mode_test.json

```diff
@@ -0,0 +1 @@
341,250.000000,0.000000]},"s":{"k":[50.000000,50.000000,0.000000]},"r":{"k":0.000000}},{"ty":"fl","nm":"Fill","hd":false,"o":{"k":100.000000},"c":{"k":[0.999456,0.986359,0.253202,1.000000]},"r":2},{"ty":"tr","nm":"Group","hd":false,"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[0.000000,0.000000,0.000000]},"s":{"k":[100.000000,100.000000,100.000000]},"r":{"k":0.000000},"o":{"k":100.000000},"sk":{"k":0.000000},"sa":{"k":0.000000}}]}]},{"nm":"Image layer","ind":2,"ty":2,"ddd":0,"ip":0.000000,"op":300.000000,"st":0.000000,"ks":{"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[-0.156402,36.758697,0.000000]},"s":{"k":[10.000000,10.000000,0.000000]},"r":{"k":0.000000},"o":{"k":100.000000}},"bm":0,"hasMask":false,"sr":1.000000,"tt":0,"hd":false,"refId":"image_0"}]}],"layers":[{"nm":"Precomp layer","ind":1,"ty":0,"ddd":0,"ip":0.000000,"op":1.000000,"st":0.000000,"ks":{"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[1.787533,133.007088,0.000000]},"s":{"k":[100.000000,100.000000,100.000000]},"r":{"k":0.000000},"o":{"k":100.000000}},"bm":0,"hasMask":false,"sr":1.000000,"tt":0,"hd":false,"refId":"precomp_10","w":100,"h":130},{"nm":"Precomp layer","ind":2,"ty":0,"ddd":0,"ip":0.000000,"op":1.000000,"st":0.000000,"ks":{"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[379.840704,-0.232576,0.000000]},"s":{"k":[100.000000,100.000000,100.000000]},"r":{"k":0.000000},"o":{"k":100.000000}},"bm":0,"hasMask":false,"sr":1.000000,"tt":0,"hd":false,"refId":"precomp_8","w":100,"h":130},{"nm":"Precomp layer","ind":3,"ty":0,"ddd":0,"ip":0.000000,"op":1.000000,"st":0.000000,"ks":{"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[283.046894,-0.115288,0.000000]},"s":{"k":[100.000000,100.000000,100.000000]},"r":{"k":0.000000},"o":{"k":100.000000}},"bm":0,"hasMask":false,"sr":1.000000,"tt":0,"hd":false,"refId":"precomp_6","w":100,"h":130},{"nm":"Precomp layer","ind":4,"ty":0,"ddd":0,"ip":0.000000,"op":1.000000,"st":0.000000,"ks":{"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[187.869433,-0.166732,0.000000]},"s":{"k":[100.000000,100.000000,100.000000]},"r":{"k":0.000000},"o":{"k":100.000000}},"bm":0,"hasMask":false,"sr":1.000000,"tt":0,"hd":false,"refId":"precomp_4","w":100,"h":130},{"nm":"Precomp layer","ind":5,"ty":0,"ddd":0,"ip":0.000000,"op":1.000000,"st":0.000000,"ks":{"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[93.608372,-0.364205,0.000000]},"s":{"k":[100.000000,100.000000,100.000000]},"r":{"k":0.000000},"o":{"k":100.000000}},"bm":0,"hasMask":false,"sr":1.000000,"tt":0,"hd":false,"refId":"precomp_2","w":100,"h":130},{"nm":"Precomp layer","ind":6,"ty":0,"ddd":0,"ip":0.000000,"op":1.000000,"st":0.000000,"ks":{"a":{"k":[0.000000,0.000000,0.000000]},"p":{"k":[0.000000,0.000000,0.000000]},"s":{"k":[100.000000,100.000000,100.000000]},"r":{"k":0.000000},"o":{"k":100.000000}},"bm":0,"hasMask":false,"sr":1.000000,"tt":0,"hd":false,"refId":"precomp_1","w":100,"h":130}],"fonts":{"list":[{"fName":"Helvetica-Regular","fFamily":"Helvetica","fStyle":"Regular","ascent":24.000000}]}}
```

Comment:
Could you move this from `Tests/Samples` to `Tests/Samples/Nonanimating/blend_mode_test.json` or `Tests/Samples/Issues/pr_1585.json`?

Reply:
Previous snapshots need to be removed?